pco_cli/Cargo.toml

```diff
@@ -22,8 +22,14 @@ better_io = { version = "0.1.0", path = "../better_io" }
 blosc-src = { version = "0.3", features = ["zstd"], optional = true }
 bytes = "1.1.0"
 clap = { version = "4.5.3", features = ["derive"] }
+half = { version = "2.4.1", features = ["std"] }
```

Comment:
Yeah we shouldn't need to add these transitive dependencies

Reply:
It won't compile without it because `pco_cli` relies on macros from `dynamics`:
```
error[E0433]: failed to resolve: use of undeclared crate or module `half`
  --> pco_cli\src\core_handlers.rs:25:3
   |
25 |   with_core_dtypes!(match_dtype)
   |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `half`
   |
   = note: this error originates in the macro `with_core_dtypes` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
Ah right, now it makes sense that we actually need it here